<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>99999</width>
    <height>99999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="3">
      <widget class="QPushButton" name="br_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Blue
Robotics</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="falcon_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{color:blue};</string>
       </property>
       <property name="text">
        <string>Falcon
Pump</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="m100_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>M100</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="maxon_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Maxon
Sensorless</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="savox_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Falcon
Savox</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>621</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Measured Velocity:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="measured_velocity">
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="motor_name">
       <property name="font">
        <font>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Falcon</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="forward_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Forward</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Set Velocity:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="set_velocity_label">
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="stop_button">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{color:red};</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="reverse_button">
       <property name="font">
        <font>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reverse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="vel_up">
         <property name="font">
          <font>
           <pointsize>50</pointsize>
          </font>
         </property>
         <property name="text">
          <string>^</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="vel_down">
         <property name="font">
          <font>
           <pointsize>50</pointsize>
          </font>
         </property>
         <property name="text">
          <string>v</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>vel_down</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>down_velocity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reverse_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_motor_state()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vel_up</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>up_velocity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forward_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_motor_state()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stop_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_motor_state()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>falcon_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_handler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>br_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_handler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m100_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_handler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxon_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_handler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savox_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>button_handler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>get_motor_velocity(int)</signal>
  <slot>button_handler()</slot>
  <slot>set_velocity(int)</slot>
  <slot>set_motor_state()</slot>
  <slot>up_velocity()</slot>
  <slot>down_velocity()</slot>
 </slots>
</ui>
